src/open_producten/locaties/models/contact.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from open_producten.utils.validators import validate_phone_number
+
+from ...utils.models import BaseModel
+from .organisatie import Organisatie
+
+
+class Contact(BaseModel):
+    organisatie = models.ForeignKey(
+        Organisatie,
+        verbose_name=_("organisatie"),
+        null=True,
+        blank=True,
+        related_name="product_contacten",
+        on_delete=models.SET_NULL,
+        help_text=_("De organisatie van het contact"),
+    )
+    voornaam = models.CharField(
+        verbose_name=_("voornaam"),
+        max_length=255,
+        help_text=_("voornaam van het contact"),
+    )
+    achternaam = models.CharField(
+        verbose_name=_("achternaam"),
+        max_length=255,
+        help_text=_("achternaam van het contact"),
+    )
+    email = models.EmailField(
+        verbose_name=_("email adres"),
+        blank=True,
+        help_text=_("email van het contact"),
+    )
+    telefoonnummer = models.CharField(
+        verbose_name=_("telefoonnummer"),
+        blank=True,
+        max_length=15,
+        validators=[validate_phone_number],
+        help_text=_("telefoonnummer van het contact"),
+    )
+    rol = models.CharField(
+        verbose_name=_("rol"),
+        blank=True,
+        max_length=100,
+        help_text=_("De rol/functie van het contact"),
+    )
+
+    class Meta:
+        verbose_name = _("Contact")
+        verbose_name_plural = _("Contacten")
+
+    def __str__(self):
+        if self.organisatie:
+            return f"{self.organisatie.naam}: {self.voornaam} {self.achternaam}"
```

Comment:
```suggestion
            return f"{self.voornaam} {self.achternaam} ({self.organisatie.naam})"
```

`John Smith (Maykin Media)` or something similar with the org being second might be a better format.

Idk what you guys decided though